markdown/blog.md

```diff
@@ -6,6 +6,14 @@ title: Blog
 
 If you'd like to contribute to the blog, please [sign up](/volunteer) and let us know!
 
+## [Blogrs](/blog/2023/02/15/blogrs)
+
+2023/02/15.
+
+This is not a blog
```

Comment:
Is this meant to be a test or new content to be added? 🤔  If so, the description here seems wrong?

Reply:
sorry, this was just a test Itsi was running over with me. Not content